Core/src/main/java/com/plotsquared/core/PlotSquared.java

```diff
@@ -1020,8 +1020,8 @@ public boolean setupPlotWorld(
             // save configuration
 
             final List<String> validArguments = Arrays
-                    .asList("s=", "size=", "g=", "gap=", "h=", "height=", "f=", "floor=", "m=", "main=",
-                            "w=", "wall=", "b=", "border="
+                    .asList("s=", "size=", "g=", "gap=", "h=", "height=", "minh=", "minheight=", "maxh=", "maxheight=",
+                            "f=", "floor=", "m=", "main=", "w=", "wall=", "b=", "border="
```

Comment:
Is that still in use? Every time I see people using these builders its down to errors. Our plot setup builder is way more robust 🤔

Reply:
Doesn't mean it should go in there to maybe be fixed